src/GovUk.Education.ExploreEducationStatistics.Data.Importer/Services/ImporterService.cs

```diff
@@ -139,8 +143,17 @@ public SubjectMeta GetMeta(DataTable table, Subject subject, StatisticsDbContext
             {
                 if (rowCount % STAGE_2_ROW_CHECK == 0)
                 {
+                    var currentStatus = await _importStatusService.GetImportStatus(releaseId, dataFileName);
+
+                    if (currentStatus.IsFinishedOrAborting())
+                    {
+                        _logger.LogInformation($"Import for {dataFileName} has finished or is being aborted, " +
+                                               $"so finishing importing Filters, Locations and Schools early");
```

Comment:
We don't import schools as far as I know.

Reply:
Reworded comment, and renamed methods